src/main/java/com/rtb/projectmanagementtool/task/TaskData.java

```diff
@@ -0,0 +1,88 @@
+package com.rtb.projectmanagementtool.task;
+
+import java.util.HashSet;
+
+/** Class containing task data. */
+public final class TaskData {
+
+  private long taskID;
+  private long projectID;
+  private String name;
+  private String description;
+  private Status status;
+  private HashSet<Long> users;
+  private HashSet<Long> subtasks;
+
+  public TaskData(long taskID, long projectID, String name, String description,
+    Status status, HashSet<Long> users, HashSet<Long> subtasks) {
```

Comment:
maybe add different constructor(s) for if some of these arguments aren't sent in